apps/web/src/containers/TradingContest/points-dashboard/hooks/use-qualifying-products-table-dropdowns.tsx

```diff
@@ -9,14 +9,13 @@ import {
   CoinsIcon,
   CoinsCircleIcon,
 } from '@notional-finance/icons';
-import { Network } from '@notional-finance/util';
 import { TokenIcon } from '@notional-finance/icons';
 import { useAllUniqueUnderlyingTokens } from '@notional-finance/notionable-hooks';
 import { useTheme } from '@mui/material';
 
 export const useQualifyingProductsTableDropdowns = () => {
   const theme = useTheme();
-  const depositTokens = useAllUniqueUnderlyingTokens([Network.arbitrum]);
+  const depositTokens = useAllUniqueUnderlyingTokens();
```

Comment:
This hook looks wrong, what would be the use case for looking across all networks?

Reply:
This doesn't matter any more because we are not using this table.